include/zephyr/bluetooth/uuid.h

```diff
@@ -2170,7 +4761,7 @@ struct bt_uuid_128 {
 /**
  *  @brief Source PAC Locations Characteristic value
  */
-#define BT_UUID_PACS_SRC_LOC_VAL 0x2BCC
+#define BT_UUID_PACS_SRC_LOC_VAL 0x2bcc
```

Comment:
I don't recall the style guide stating whether hex values should be in capital or not, but I would suggest not to change all to lower case to reduce the number of lines you modify in this PR. 

There's not really any reason to change them

Reply:
Idea was just to have the consistent representation in the whole file. If these 73 out of 3411 changes mattes, I can revert them.

Reply:
Personally I'd avoid modifying existing UUIDs for simplicity :) 

Won't block the PR based on this, but I think it would be better to leave them as is